lib/dom_dsl_core.ml

```diff
@@ -9,7 +9,7 @@ module Prop = struct
 
   let int key (value : int) = any key value
 
-  let float key (value : float) = any key value
+  let float_ key (value : float) = any key value
```

Comment:
I updated this value of `float_` to not collide. But the other one in `Core` is left as is, as in that case collisions should not happen:

https://github.com/jchavarri/jsoo-react/blob/7767fc1a62f0530ea2e62ca50ff43672f50e522a/lib/dom_dsl_core.ml#L45